meeting-notes/2018/2018-12-10--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,218 @@
+    - Link fix in js-ipfs core files readme [411](https://github.com/ipfs/interface-ipfs-core/pull/411)
+- blocking:
+    - none
+- next:
+
+	- Finish verify files and pre-generating them based on config [33](https://github.com/ipfs/benchmarks/pull/53)
+    - Fix multi-peer transfer bug
+    - Cat files (websocket, mplex) - Node.js
+    - Cat files (websocket, mplex, secio) js01234->js5 - Node.js
+    - Node initialization - Browser
+    - Add files - Browser
+    - Cat files (websocket, mplex, secio) js0 -> js1 - Browser
+
+
+@hugomrdias
+- Done:
+    - feat: add package.json linter [#ipfs/aegir/304](https://github.com/ipfs/aegir/pull/304)
+    - fix: make babel and webpack safer and faster [#ipfs/aegir/305](https://github.com/ipfs/aegir/pull/305)
+    - fix: fix repo lock and buffer api [#ipfs/js-ipfs-repo/185](https://github.com/ipfs/js-ipfs-repo/pull/185)
+    - feat: make experimental default and improve test cmd [#ipfs/aegir/306](https://github.com/ipfs/aegir/pull/306)
+    - bundle size
+        - fix: clean up, bundle size reduction [https://github.com/libp2p/js-libp2p-crypto/pull/134](https://github.com/libp2p/js-libp2p-crypto/pull/134)
+        - fix: clean repo and bundle size reduction [https://github.com/libp2p/js-peer-id/pull/90](https://github.com/libp2p/js-peer-id/pull/90)
+        - Feat/bundle size [https://github.com/multiformats/js-multiaddr/pull/75](https://github.com/multiformats/js-multiaddr/pull/75)
+        - chore: update bl [https://github.com/mafintosh/tar-stream/pull/98](https://github.com/mafintosh/tar-stream/pull/98)
+        - feat: clean repo and bundle size reduction [https://github.com/libp2p/js-peer-info/pull/71](https://github.com/libp2p/js-peer-info/pull/71)
+        - use jsbi instead of bignumber [https://github.com/dignifiedquire/borc/issues/37](https://github.com/dignifiedquire/borc/issues/37)
+        - Successfully published iso-url@0.1.0
+        - forked stream-http [https://github.com/hugomrdias/stream-http](https://github.com/hugomrdias/stream-http)
+
+- Blocked
+    - Fix non disposable daemon init/start and attach to running daemons #308 [https://github.com/ipfs/js-ipfsd-ctl/pull/308](https://github.com/ipfs/js-ipfsd-ctl/pull/308)
+    - fix: pass config to init #303 [https://github.com/ipfs/js-ipfsd-ctl/pull/303](https://github.com/ipfs/js-ipfsd-ctl/pull/303)
+    - [WIP] feat: Try out CircleCI's new infrastructure for test & release flow #1669
+    - CI for js-ipfs #442 [https://github.com/ipfs/infrastructure/issues/442](https://github.com/ipfs/infrastructure/issues/442)
+- Next
+    - finish bundle size work on ipfs and ipfs-api
+    - rework the error proposal with support for wrapping
+    - multihashing, libp2p-mplex async iterators
+    - make the stream-http fork a package to fix the streams in sw issue
+    - more reviews :)
+
+@achingbrain
+ - Done:
+   - Did a talk at opo.js
```

Comment:
Was it recorded? I'd like to see it.

Reply:
It was and it will be published :)  Follow https://medium.com/oponetwork for the release //cc @marcooliveira @satazor

Reply:
It got posted!  It's here: https://www.youtube.com/watch?v=umNL2VkIHe8